src/test/resources/standalone/config/router-config.json

```diff
@@ -2,17 +2,17 @@
   "configVersion": "v2.2.0-EN000121",
   "streetRoutingTimeout": "5s",
   "routingDefaults": {
-    "walkSpeed": 1.3,
+    "walkSpeed": 1.33,
```

Comment:
I think we want these changes - look at it as clen-up, not relevant, but indeed welcome.

Reply:
It is required due to this change:

https://github.com/opentripplanner/OpenTripPlanner/pull/4977/commits/a95253a02cab5f483cd4279856ce3428b817fde6#diff-4694584dc4047e7a37434a4b1dcc131e4c40470f737bf21eb222b7180de2fdf8L61-R61

`router-config.json` is parsed when creating the documentation. Since the values in `RouteRequest` are now copied from the _default request_ (due to using `clone()`) parsing  `transferCacheRequests` modifies the stored default values for the documentation. `routingDefaults` was updated to the default values due to this.

Reply:
It seems the changes are not needed anymore, I've removed them.